tools/pbmm2_align.cwl

```diff
@@ -117,8 +117,8 @@ inputs:
   collapse_homopolymers: { type: 'boolean?', inputBinding: { prefix: "--collapse-homopolymers", position: 1 }, doc: "Collapse homopolymers in reads and reference." }
 
   # Control
-  cores: { type: 'int?', default: 16, inputBinding: { prefix: "--num-threads", position: 1 }, doc: "Number of threads to use" }
-  ram: { type: 'int?', default: 32, doc: "RAM (in GB) to use" }
+  cpu: { type: 'int?', default: 36, inputBinding: { prefix: "--num-threads", position: 1 }, doc: "Number of threads to use" }
```

Comment:
why 36 and not 32 as defaults for cpu and ram?

Reply:
c5.9x is 36. Figured I should just give it all the cores on a machine when it will be running alone.